Translations/pt_BR.po

```diff
@@ -6530,20 +6484,16 @@ msgid "Some Magic Immunities"
 msgstr "Algumas Imunidades à Magia"
 
 #: Source/mpq/mpq_writer.cpp:161
-#, fuzzy
-#| msgid "Failed to open stash archive for writing."
 msgid "Failed to open archive for writing."
-msgstr "Não foi possível abrir o arquivo do armazém para escrita."
+msgstr "Não foi possível abrir o arquivo para escrita."
```

Comment:
Original should probably change from "archive" to the much more normal "file" here.

Reply:
I agree but it's out of scope of this PR.

Reply:
Yeah, this was more for @AJenbo to consider for a separate PR. Same for similar comments.